### PR TITLE
add images

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
Good work Harsh!  looks good to me ✅

**Comment by vaibhav18matere** [comment]
looks good to me !

File: images/images.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            </div>
+            <div class="doc-img">
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2509%2509%2509%253Cdiv%2520class%253D%2522fluid_img%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520%2520class%253D%2522round_img%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+                style="width: 454px; height: 316px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (vaibhav18matere) [inline images/images.html:53]: you can make width to 100%

Comment (harshkotkar5557) [reply images/images.html:53]: okay Done

**Pull request merged**